### PR TITLE
feat: use `nw` flag conditionally

### LINKED ISSUE #4: remove the `--nw` flag once new scully version is released
Hey,

with one of the latest commits, the scully option `--nw` has been removed.
The default behaviour has been changed back to run just one and use `--watch` to explicitely start the watch mode.

https://github.com/scullyio/scully/commit/10950b5771fd4d82b5b4b2c31df7737523dad044

Probably there should be an version check to auto add `--nw` when using scully until the current version  and leaving the flag when using a newer version.

### PR DESCRIPTION
**Pull request by d-koppenhagen**
fix #4

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -46,7 +47,18 @@ async function run() {
...
 
-    await exec.exec(`${pkgManager} run scully -- --nw ${scullyArgs}`, []);
+    // determine the scully version
+    const packageJsonRaw = await fs.readFile('./package.json', 'utf8');
+    const packageJsonParsed = JSON.parse(packageJsonRaw);
+    const scullyVersion = packageJsonParsed.dependencies['@scullyio/scully']
+      .replace('^', '')
+      .replace('~', '');
+    // add the `--nw` flag if scully version is below or equal `0.0.85`
+    if (scullyVersion <= '0.0.85') {
```
Comment (cmgriffing) [inline index.js:57]: How would you feel about using node-semver for this version comparison instead?

I'm just fuzzy on the possible edge cases that using raw string comparison would have.

File: index.js [hunk 2]
```diff
@@ -46,7 +47,18 @@ async function run() {
     await exec.exec(`${pkgManager} run build ${buildArgs}`, []);
     console.log("Finished building your site.");
 
-    await exec.exec(`${pkgManager} run scully -- --nw ${scullyArgs}`, []);
+    // determine the scully version
+    const packageJsonRaw = await fs.readFile('./package.json', 'utf8');
```
Comment (cmgriffing) [inline index.js:51]: Maybe we could parse or look for the package-lock/yarn.lock file?

Then the version value there would not need to be scrubbed of `~` or `^`.

File: index.js (hunk 1 above)
Comment (d-koppenhagen) [reply index.js:57]: sure, makes much more sense

File: index.js (hunk 2 above)
Comment (d-koppenhagen) [reply index.js:51]: ah yes, that would probably a good idea. Youst thinking about if it makes sense when one does not check in the lock files? I think as of `npm install` in the CLI, an old version could be used?

Comment (d-koppenhagen) [reply index.js:51]: ah no. `npm i` / `yarn i` should update the lock file before.

Comment (cmgriffing) [reply index.js:51]: Oh good call. You had me wondering too lol.

**Comment by d-koppenhagen** [comment]
@cmgriffing I think this should work now. Are you able to test it? I could verify it later on my own scully site using the new workflow version

**Comment by cmgriffing** [comment]
Awesome. Everything looks good. I can have time to test it out more a bit later today, I think.

**Comment by d-koppenhagen** [comment]
hey @cmgriffing did you find some time for testing?

**Comment by cmgriffing** [comment]
Sorry I got blocked figuring out how to test it before merging.

I think I have to just merge it and do an experimental version on the marketplace. Then I can test it with one of my side projects.

You may be able to do the same just to be sure.

I'm going to brainstorm a better way of containing that kind of testing to this repo in the future, if I can.

**Comment by d-koppenhagen** [comment]
oh yes, I was wondering too how to test it or even write local tests for the workflow. You can keep me posted :)

**Pull request merged**